src/Run/CommandLine.cs

```diff
@@ -1813,7 +1825,7 @@ private static string GetEntryAssemblyName()
         private bool _noDashOnParameterSets = true;
         private bool _noSpaceOnQualifierValues;
         private string[] _parameterSetsWhereQualifiersMustBeFirst;
-        private bool _qualifiersUseOnlyDash = true;
+        private bool _qualifiersUseOnlyDash = false;
```

Comment:
I'm not sure I understand what you mean. In either case you can make run.exe have whichever default the tool needs.